src/assets/css/_main.scss

```diff
@@ -1578,7 +1580,7 @@ a.tile-block-link {
   top: calc(100% + 1px);
   width: 100%;
   height: 1px;
-  background: rgba(0, 0, 0, 0.85);
+  background: rgba(0, 0, 0, 0.85%);
```

Comment:
`0.85` is different to `0.85%` isn't it?  (0.85 is 85%)

Reply:
Good catch :) Fix incoming for all of them.